src/redux/books/books.js

```diff
@@ -1,23 +1,25 @@
 // Actions
-const ADD = 'bookstore/book/ADD';
-const REMOVE = 'bookstore/book/remove';
+const Actions = {
+  ADD_BOOK: 'bookstore/book/ADD',
+  REMOVE_BOOK: 'bookstore/book/remove',
```

Comment:
Excellent ✔ The constant follows the naming convention! As I was not clear enough, I will approve your changes even though we need to change one more thing 🤓 

Your type action:
       
    REMOVE_BOOK: 'bookstore/book/remove',

Intended:

    REMOVE_BOOK: 'bookstore/book/REMOVE',

Reply:
Hi @mateo951 
Thanks for a kind review. I'll modify the naming for the REMOVE action as requested.